Client/mods/deathmatch/logic/CClientRadarAreaManager.cpp

```diff
@@ -79,6 +80,16 @@ void CClientRadarAreaManager::SetDimension(unsigned short usDimension)
     m_usDimension = usDimension;
 }
 
+void CClientRadarAreaManager::SetInterior(unsigned char ucInterior)
+{
+    for (CClientRadarArea *pArea : m_List)
```

Comment:
Check if you have to update the interior here: `if (m_ucInterior != ucInterior)`